Plugins/BenchmarkTool/BenchmarkTool+Operations.swift

```diff
@@ -96,16 +96,64 @@ extension BenchmarkTool {
                 }
 
                 let currentBaseline = benchmarkBaselines[0]
-                let baselineName = baseline[0] == "default" ? "Current baseline" : baseline[0]
-                let comparingBaselineName = compare ?? "unknown"
 
                 prettyPrintDelta(currentBaseline: currentBaseline, baseline: comparisonBaseline)
 
-                if currentBaseline.betterResultsOrEqual(than: comparisonBaseline, printOutput: true) {
+                return
+            }
+
+            if let checkBaseline {
+                guard benchmarkBaselines.count > 0 else {
+                    print("Only had \(benchmarkBaselines.count) baselines available, can't check.")
```

Comment:
But the only possible value is 0, maybe just say `"No baselines available, cat't check"`?